apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/metadata.json

```diff
@@ -6,7 +6,7 @@
   "kind" : "instance",
   "software" : {
     "name" : "Blue Button API: Direct",
-    "version" : "2.167.0-SNAPSHOT"
+    "version" : "IGNORED_FIELD"
```

Comment:
As a bonus to fixing the failures, I included this in the ignored field list so we don't get these annoying diffs all the time.